src/encoded/static/scss/encoded/modules/_pipeline.scss

```diff
@@ -1,3 +1,6 @@
+$graph-download-font: 'Helvetica Neue', sans-serif;
```

Comment:
Not to self-  Mada not used used due to potential issues with rendering a non-built in library